starcheck/utils.py

```diff
@@ -297,12 +310,27 @@ def imposter_offset(cand_mag, imposter_mag):
         spoil_counts = mag_to_count_rate(imposter_mag)
         return spoil_counts * 3 * 5 / (spoil_counts + cand_counts)
 
+    # Get the effective t_ccd for use for the fid lights and some guide stars.
+    eff_t_ccd = get_effective_t_ccd(t_ccd)
+
+    # Get the t_ccd bonus for n=dyn_bgd_n_faint of the guide stars.
+    guide_mags = []
+    guide_idxs = []
+    for idx, mag, ctype in zip(idxs, mags, types):
+        if ctype in ["BOT", "GUI"]:
+            guide_mags.append(mag)
+            guide_idxs.append(idx)
+    guide_t_ccds = _t_ccds_bonus(guide_mags, t_ccd, date)
```

Comment:
It's not really a thing any more, but should the `t_ccd` here instead be `eff_t_ccd`? Otherwise the guide stars are getting an extra bonus of not having the penalty limit applied.

Reply:
_t_ccds_bonus applies the penalty, so this is logically OK, but it looks like I should remove that from _t_ccds_bonus for clarity and instead pass the effective t_ccd.

Reply:
I ended up moving the effective_t_ccds back into guide_count instead of including it in apply_t_ccds_bonus.